internal/cli/alerts/settings/validator.go

```diff
@@ -91,8 +91,8 @@ func validateOpsGenie(opts *ConfigOpts) error {
 }
 
 func validatePagerDuty(opts *ConfigOpts) error {
-	if opts.apiKey == "" {
-		return fmt.Errorf("--%s is required when --%s is PAGER_DUTY", flag.APIKey, flag.NotificationType)
+	if opts.notificationServiceKey == "" || opts.notificationRegion == "" {
```

Comment:
Very happy about this change at least. If you can confirm that we have any sign-off we need for this, and delete the other file, LG.

Reply:
This is more of a bug fix since this was not working as intended before. The apiKey was piped into nothing before and we did not surface the validation on region to the command line before (even though the region is required). But if you think we need sign-off I can ask Shalini 👍

Reply:
To be honest, I meant if there were other upstreams, but if that's not the case, then 👍